products/couchbase-server.md

```diff
@@ -22,17 +22,17 @@ releases:
 -   releaseCycle: "7.1"
     eol: false
     latest: "7.1.4"
-    latestReleaseDate: 2023-03-16
+    latestReleaseDate: 2023-04-18
```

Comment:
Wrong update due to the auto configuration. We should find another method to auto-update releases.